docs/docs/1-getting-started/2-installation/index.md

```diff
@@ -18,6 +21,12 @@ DynamoDB-Toolbox has `@aws-sdk/client-dynamodb` and `@aws-sdk/lib-dynamodb` as p
 # npm
 npm install @aws-sdk/client-dynamodb @aws-sdk/lib-dynamodb
 
+# pnpm
+pnpm add @aws-sdk/client-dynamodb @aws-sdk/lib-dynamodb
+
 # yarn
 yarn add @aws-sdk/client-dynamodb @aws-sdk/lib-dynamodb
 ```
+
+:::info
```

Comment:
This is not the correct way to do admonitions: https://docusaurus.io/docs/markdown-features/admonitions

```
:::info

Your text here.

:::
```